src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+function CloneElementOfRow(row) {
```

Comment:
```suggestion
function CloneElementOfRows(rows, from, to) {
```
Pass an array here and loop through it.